server/Makefile

```diff
@@ -0,0 +1,22 @@
+$(shell cargo install default-target)
```

Comment:
i don't like running things outside of makefile targets - can we avoid this?

Reply:
This is here because I tried conditionally setting variables within the target, but the variable name kept getting interpreted as a command. I'm sure there's some way around it, if I end up finding it, I will include in a follow-up PR.